src/components/Accordion/Accordion.tsx

```diff
@@ -36,6 +38,19 @@ const AriaAccordionItem: FC<AriaAccordionItemProps> = ({ item, state, header })
     const { buttonProps, regionProps } = useAccordionItem({ item }, state, triggerRef);
     const isOpen = state.expandedKeys.has(item.key) && item.props.children;
     const { isFocusVisible, focusProps } = useFocusRing();
+    const [isActive, setIsActive] = useState(false);
+    const isActiveByDefault = header.active;
```

Comment:
Personally, I'd find it more intuitive if you combine this with the `active` prop:
```suggestion
    const [isActive, setIsActive] = useState(header.active);
```